eyelid-match-ops/src/primitives/poly/test/inv.rs

```diff
@@ -0,0 +1,29 @@
+//! Tests for polynomial inverse.
+
+use super::super::*;
+use rand::Rng;
+
+/// This sampling is similar to what will be necessary for YASHE KeyGen
```

Comment:
If this sampling function is going to be used in production key generation, can it move to `poly/modular_poly/inverse.rs`?

(If it’s going to be used in production, it doesn’t need to be benchmark and test only.)

Reply:
Yes, I was expecting to move it in the near future, then it makes sense to `inverse.rs` now.